src/config.js

```diff
@@ -178,6 +178,14 @@ export const WMS_TILE_SIZE = 512 // px
 
 export const VIEW_MIN_RESOLUTION = 0.1 // meters/pixel
 
+/**
+ * Map default max resolution as it is set in mf-chsdi. When the layer has this as a max resolution,
+ * the layersConfig endpoint does not give the resolutions available.
+ * https://github.com/geoadmin/mf-chsdi3/blob/0236814544a6bf2df86598889b81ee4023494325/chsdi/models/bod.py#L119-L123
```

Comment:
@pakb @rebert @ltclm what does it mean when a layer don't give the resolutions in mf-chsdi3 ? Why do we have this use case at all ? Or should all layers define their resolutions ? From what I've seen/tested Geojson/GPX/KML layers don't require the max resolution, but WMS and WMTS layer do. What about Vector layer is it the same as KML layer I guess ?

Reply:
`minResolution/maxResolution` settings in such layers are used in mf-geoadmin3 for the scale dependant request of different layers from the backend, this is used in so called aggregated layers. this can be combinations of type wms/wms or wmts/wms.

```json
"ch.swisstopo-vd.geometa-gemeinde_wms": {
"gutter": 15,
"timeEnabled": false,
"searchable": false,
"type": "wms",
"tooltip": false,
"minResolution": 0,
"topics": "api,cadastre,ech,inspire,service-wms,swisstopo",
"background": false,
"maxResolution": 20,
"wmsLayers": "ch.swisstopo-vd.geometa-gemeinde",
"label": "ch.swisstopo-vd.geometa-gemeinde_wms",
"parentLayerId": "ch.swisstopo-vd.geometa-gemeinde",
"attribution": "AV",
"chargeable": false,
"hasLegend": true,
"serverLayerName": "ch.swisstopo-vd.geometa-gemeinde",
"highlightable": true,
"format": "png",
"singleTile": false,
"opacity": 0.75,
"wmsUrl": "https://wms.geo.admin.ch/",
"attributionUrl": "https://www.cadastre.ch/de/home.html"
},
"ch.swisstopo-vd.geometa-gemeinde_wmstile": {
"timeEnabled": false,
"searchable": false,
"type": "wms",
"tooltip": false,
"minResolution": 20,
"topics": "api,cadastre,ech,inspire,service-wms,swisstopo",
"background": false,
"wmsLayers": "ch.swisstopo-vd.geometa-gemeinde",
"label": "ch.swisstopo-vd.geometa-gemeinde_wmstile",
"parentLayerId": "ch.swisstopo-vd.geometa-gemeinde",
"attribution": "AV",
"chargeable": false,
"hasLegend": true,
"serverLayerName": "ch.swisstopo-vd.geometa-gemeinde",
"highlightable": true,
"format": "png",
"singleTile": false,
"opacity": 0.75,
"wmsUrl": "https://wms.geo.admin.ch/",
"attributionUrl": "https://www.cadastre.ch/de/home.html"
},

```

Reply:
@ltclm thanks for the insight. We checked with @ltshb  and it seems aggregate layers are not showing at all in the new mapviewer, and you were right regarding wms layers. I'll correct this PR to only affect  WMTS layers, then I'll investigate our aggregate layers problem.

Reply:
If I remember what I've discussed with @ltclm : only resolution that aren't "default" are given in the config, the one you have set as the default as constant is to be used otherwise